kotlin/feature-layer-selection/README.md

```diff
@@ -0,0 +1,34 @@
+# Feature layer selection
+
+Select features in a feature layer.
+
+![Image of feature layer selection](feature-layer-selection.png)
+
+## Use case
+
+Selecting features, whether by query or identify, can be an important step both in editing data and visualizing results. One possible use case would be to query a feature layer containing street furniture. A query might look for type "bench" and return a list of bench features contained in the features with an attribute of type bench. These might be selected for further editing (see FeatureQueryResult) or may just be highlighted visually.
```

Comment:
"whether by query or identify" probably is worded correctly according to api, but it sounds pretty strange. Perhaps identification?

Reply:
I know what you mean, but since it makes sense from an API perspective, let's leave as is as I think it's clear enough :)